src/annotator-content-analysis.js

```diff
@@ -80,6 +80,19 @@ function annotatorContentAnalysis(options){
 
         })
 
+        $('.commit-question').on('click', (e) => {
```

Comment:
ok - so I understand you correctly, you're suggesting to not put 
```
<button class="commit-question" id="question-{{element.question_id}}"type="button">next</button>
``` 

in `question.handlebars`, but to put this kind of thing in `topic.handlebars`? and then grab the dependency from the answer...